PaddleHQ/transaction-search
# Fix casing for one-time

### Checklist

### What has been done:
`one-time` has been changed to `oneTime` to help @amypaddle parse the response. As this is a constant value, I don't object to forcing a case here.

### How to test:
1. Make a request for a non-subscription transaction
1. Confirm that "one-time" is now "oneTime"
